Source/Core/AudioCommon/OpenALStream.h

```diff
@@ -68,9 +68,9 @@ class OpenALStream final : public SoundStream
   Common::Flag m_run_thread;
 
   std::vector<short> m_realtime_buffer;
-  std::array<ALuint, OAL_BUFFERS> m_buffers;
-  ALuint m_source;
-  ALfloat m_volume;
+  std::array<ALuint, OAL_BUFFERS> m_buffers{};
+  ALuint m_source = 0;
+  ALfloat m_volume = 1;
```

Comment:
I know nothing about this system.  Is 1 intentional here?

Reply:
It's used here and here:

https://github.com/dolphin-emu/dolphin/blob/05f86b9f5b6e7f65aaefcc29e348cc2628bed34f/Source/Core/AudioCommon/OpenALStream.cpp#L259-L260

https://github.com/dolphin-emu/dolphin/blob/05f86b9f5b6e7f65aaefcc29e348cc2628bed34f/Source/Core/AudioCommon/OpenALStream.cpp#L147-L153

And the docs say 1.0 leaves volume unchanged:

https://github.com/dolphin-emu/dolphin/blob/05f86b9f5b6e7f65aaefcc29e348cc2628bed34f/Externals/OpenAL/include/al.h#L184-L196